WooCommerce/Classes/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel.swift

```diff
@@ -24,4 +28,101 @@ private extension DefaultProductFormTableViewModel {
             .description(description: product.trimmedFullDescription)
         ]
     }
+
+    func settingsRows(product: Product) -> [ProductFormSection.SettingsRow] {
+        return [
+            .price(viewModel: priceSettingsRow(product: product)),
+            .shipping(viewModel: shippingSettingsRow(product: product)),
+            .inventory(viewModel: inventorySettingsRow(product: product))
+        ]
+    }
+}
+
+private extension DefaultProductFormTableViewModel {
+    func priceSettingsRow(product: Product) -> ProductFormSection.SettingsRow.ViewModel {
+        let icon = UIImage.priceImage.applyProductFormSettingsStyle()
+        let title = Constants.priceSettingsTitle
+
+        var priceDetails = [String]()
+
+        // Regular price and sale price are both available only when a sale price is set.
+        if let regularPrice = product.regularPrice, !regularPrice.isEmpty,
+            let salePrice = product.salePrice, !salePrice.isEmpty {
+            let formattedRegularPrice = currencyFormatter.formatAmount(regularPrice, with: currency) ?? ""
+            let formattedSalePrice = currencyFormatter.formatAmount(salePrice, with: currency) ?? ""
+            priceDetails.append(String.localizedStringWithFormat(Constants.regularPriceFormat, formattedRegularPrice))
+            priceDetails.append(String.localizedStringWithFormat(Constants.salePriceFormat, formattedSalePrice))
```

Comment:
As far as I know, I'm not sure that the use of variables within the localizedStringWithFormat is taken from the script that manages the localization 🤔

Reply:
The format is a localizable string! In this example, it's using the format string `Constants.salePriceFormat` which is `NSLocalizedString("Sale price: %@", comment: "Format of the sale price on the Price Settings row")` so this format can be translated into different languages with the variable `%@` 🙂